pycbc/waveform/parameters.py

```diff
@@ -417,32 +417,35 @@ def docstr(self, prefix='', include_label=True):
                 dtype=float, default=0., label=r"$\delta$",
                 description="Mean anomaly of the periastron (rad).")
 tc = Parameter("tc",
-                dtype=float, default=None, label=r"$t_c$ (s)",
-                description="Coalescence time (s).")
+               dtype=float, default=None, label=r"$t_c$ (s)",
+               description="Coalescence time (s) is the time when a GW "
+                           "reaches the origin of a certain coordinate system.")
 delta_tc = Parameter("delta_tc", dtype=float,
                      label=r"$\Delta t_c~(\rm{s})$",
-                     description="Coalesence time offset.")
+                     description="Coalesence time offset in " +
+                     "a certain coordinate system.")
```

Comment:
Drop 'in a certain coordinate system'. I don't think that adds any clarification.

Reply:
@ahnitz Yeah, that makes sense. I have removed it.